molecule/default/install-deb.yml

```diff
@@ -3,6 +3,7 @@
     name:
       - fontconfig
       - openjdk-11-jre
```

Comment:
Debian 12 doesn't ship openjdk 11:
`E: Package 'openjdk-11-jre' has no installation candidate`

Either the task needs splitting with a version conditional or a different playbook file created for debian 12